src/bp/sdk/botpress.d.ts

```diff
@@ -5,6 +5,7 @@
  * Please let us know in our official Github Repo!
  */
 declare module 'botpress/sdk' {
+  import { FormData } from 'common/typings'
```

Comment:
When you need typings in the SDK, please move them from common to the SDK. The code editor can't import them, and we try to keep the SDK declaration file as a standalone file.